lib/middleware.ts

```diff
@@ -113,6 +113,97 @@ var defaultHandlers = [
   () => ({title: "request-finished"}),
 ];
 
+const defaultContextHandlers = [
+  (req) => ({"request-id": req.headers["x-request-id"]}),
+  (req) => ({"session-id": req.sessionID}),
```

Comment:
Most of our services don't have `x-request-id` either. I don't think we should have any defaults other than what's already built into the middleware